.rubocop.yml

```diff
@@ -37,8 +37,15 @@ Metrics/AbcSize:
   Exclude:
     - 'spec/**/*'
 
+Metrics/ClassLength:
+  Exclude:
+    - 'app/form_builders/mps_form_builder.rb'
```

Comment:
I've left a note in the first commit explaining why I decided to exclude this file from Rubocop.